src/services/screenOptions.tsx

```diff
@@ -0,0 +1,59 @@
+import { NavigationProp } from '@react-navigation/native'
```

Comment:
Good idea to move this to a own file

Why have you decided to put this file not into navigation directory? 

just because it catched my eyes: I think we should think about good naming of services.